Libraries/Utilities/Platform.ios.js

```diff
@@ -14,6 +14,7 @@
 
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => obj.ios,
```

Comment:
I will play around internally tomorrow when we start integrating that and see how Flow behaves. Just started using it recently, so I'd better check first.